students/Gabe_Lamberth/lesson09/run_mailroom.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python3
+from mailroom5 import *
```

Comment:
`*` imports are generally frowned upon as no one will know what functions belong to it.  be explicit about what you are importing.